\documentclass{klmr-cv}

\setotherlanguage{french}

\author{Konrad Rudolph}
\email{konrad.rudolph@gmail.com}
\website{klmr.me}

\newcommand*\csharp{C\#}
\newcommand*\cpp{C++}
\newcommand*\macOS{\mbox{macOS}}

\begin{document}

\maketitle

\begin{sidebar}
    \section{Degrees}

    \subsection{University of Cambridge\date{2016}}
    \entry{PhD}
    \details{Bioinformatics}

    \subsection{Freie Universit√§t Berlin\date{2011}}
    \entry{Master of Science}
    \details{Bioinformatics}

    \subsection{Freie Universit√§t Berlin\date{2008}}
    \entry{Bachelor of Science}
    \details{\subsubsection{Major:} bioinformatics}
    \details{\subsubsection{Minor:} physics}

    \section{Profiles}

    \subsection{GitHub}
    \details{\href{http://github.com/klmr}{klmr}}

    \subsection{ORCID}
    \details{\href{http://orcid.org/0000-0002-9866-7051}{0000-0002-9866-7051}}

    \subsection{Stack Overflow}
    \details{\href{http://stackoverflow.com/users/1968/konrad-rudolph}{Konrad Rudolph}}

    \section{Skills}

    \subsection{Languages}
    \details{Fluency in written \& spoken English, German, and French}

    \subsection{Analysis}
    \details{%
        \begin{minilist}
            \item Data exploration
            \item data visualisation
            \item statistical modelling
            \item RNA-seq
            \item ChIP-seq
            \item motif analysis
        \end{minilist}}

    \subsection{Programming}
    \details{%
        \begin{minilist}
            \item \cpp{}
            \item C
            \item R
            \item Python
            \item Perl
            \item Bash
            \item Make
            \item \csharp{}
            \item Java
            \item PHP
            \item VB
            \item JavaScript
            \item Ruby
            \item ‚Ä¶
        \end{minilist}}

    \subsection{Tools}
    \details{%
        \begin{minilist}
            \item Version control
            \item unit testing
            \item reproducible research
            \item literate programming
            \item lexical/semantic analysis
            \item Unix
            \item Windows
            \item \macOS{}
        \end{minilist}}

    \subsection{Management}
    \details{%
        \begin{minilist}
            \item Software development team lead
            \item EMBL PhD Symposium organising committee
            \item EMBL Bioinformatics Workshop coordination
        \end{minilist}}
\end{sidebar}

\section{Experience}

\subsection{University of Cambridge}
\entry{Postdoctoral research associate\date{May 2016--}}
\details{\subsubsection{Group leader:} Prof Dr Eric Miska}

\subsection{EMBL-EBI}
\entry{Postdoctoral fellow\date{Oct 2015--Mar 2016}}
\details{\subsubsection{Group leader:} Dr John Marioni}

\entry{Predoctoral fellow\date{Oct 2011--Sep 2015}}
\details{\subsubsection{Thesis:} Investigating the link between tRNA and mRNA
    abundance in mammals}
\details{\subsubsection{Group leader:} Dr John Marioni}

\subsection{Independent consultancy}
\entry{Developer\date{Jun 2011--Aug 2011}}
\details{Integration of FPGA kernel with \cpp{} library}

\subsection{Illumina Inc.}
\entry{Research associate (intern)\date{Oct 2008--Jan 2009}}
\details{Implementation of short-read mapping on GPGPUs with Nvidia CUDA}

\subsection{Freie Universit√§t Berlin}
\entry{Master project student\date{2010--2011}}
\details{\subsubsection{Thesis:} Generic parallelisation of a sequence analysis
    library}
\details{\subsubsection{Group leader:} Prof Dr Knut Reinert}

\entry{Tutor\date{2008--2011}}

\entry{Bachelor project student\date{2008}}
\details{\subsubsection{Thesis:} Implementation of a read mapping tool based on the pigeon-hole
    principle}
\details{\subsubsection{Group leader:} Prof Dr Knut Reinert}

\subsection{ITosa}
\entry{Full stack developer\date{Jan 2007--Dec 2007}}
\details{\csharp{} Windows application; PHP/JavaScript/HTML web application \&
    database frontend}

\section{Teaching}

\subsection{University of Cambridge}
\entry{smallRNA sequencing\date{2017}}

\entry{NST Part II BBS Bioinformatics minor\date{2017}}

\entry{Next generation sequencing course\date{2013--2015}}
\details{High-throughput sequencing techniques, with focus on ChIP-seq \&
    RNA-seq}

\subsection{EMBL-EBI\date{2011--2015}}
\entry{Bioinformatics workshop}
\details{%
    \begin{minilist}
        \item Bash scripting
        \item Unix
        \item Git
        \item R
        \item \LaTeX{}
    \end{minilist}}

\subsection{Freie Universit√§t Berlin\date{2008--2011}}
\details{%
    \begin{minilist}
        \item Algorithms 101
        \item Algorithms 102
        \item Algorithms in bioinformatics
        \item Database systems
        \item \cpp{}
    \end{minilist}}

\section{Publications}

% Bib(La)TeX does not cope at all with shared first authors (what the heck!).
% Hence we simply write the references here directly. ü§∑  ¬Ø\_(„ÉÑ)_/¬Ø

\begin{bibliography}
    \bibitem{
        Alper Akay,
        Tom√°s Di Domenico,
        Kin M Suen,
        Amena Nabih,
        Guillermo E Parada,
        Mark Larance,
        Ragini Medhi,
        Ahmet C Berkyurek,
        Xinlian Zhang,
        Christopher J Wedeles,
        \myself{Konrad L M Rudolph},
        Jan Engelhardt,
        Martin Hemberg,
        Ping Ma,
        Angus I Lamond,
        Julie M Claycomb,
        Eric A Miska}
    {Small RNA pathways require the helicase Aquarius/EMB0-4 to heritably
        silence transcription}
    {Developmental Cell}
    {accepted}
    {}

    \bibitem{
        Pieter van Delft,
        Alper Akay,
        Sabrina M Huber,
        Christoph Bueschl,
        \myself{Konrad L M Rudolph},
        Tom√°s Di Domenico,
        Rainer Schuhmacher,
        Eric A Miska,
        Shankar Balasubramanian
    }
    {The profile and dynamics of RNA modifications in animals}
    {ChemBioChem}
    {Apr 2017}
    {\href{http://dx.doi.org/10.1002/cbic.201700093}{DOI:
        10.1002/cbic.201700093}}

    \bibitem{
        Bianca M Schmitt*,
        \myself{Konrad L M Rudolph*},
        Panagiota Karagianni,
        Nuno A Fonseca,
        Robert J White,
        Iannis Talianidis,
        Duncan T Odom,
        John C Marioni,
        Claudia Kutter
    }
    {High-resolution mapping of transcriptional dynamics across tissue
        development reveals a stable mRNA--tRNA interface}
    {Genome Research}
    {Aug 2014}
    {\href{http://dx.doi.org/10.1101/gr.176784.114}{DOI: 10.1101/gr.176784.114}}

    \bibitem{
        \myself{Konrad L M Rudolph*},
        Bianca M Schmitt*,
        Diego Villar,
        Robert J White,
        John C Marioni,
        Claudia Kutter,
        Duncan T Odom
    }
    {Codon-driven translational efficiency is stable across diverse mammalian
        cell states}
    {PLOS Genetics}
    {May 2016}
    {\href{http://dx.doi.org/10.1371/journal.pgen.1006024}{DOI:
        10.1371/journal.pgen.1006024}}

    \bibitem{
        Anja Thormann,
        \myself{Konrad L M Rudolph},
        Ina Koch
    }
    {TInA (T-Invariant Analysis): a tool box for exploring pathways in
        biochemical systems at steady state}
    {Abstract book of CGB 2009}
    {Sep 2009}
    {}
\end{bibliography}

\section{Presentations}

\subsection{Invited talks}
\entry{Cambridge Epigenetics Symposium\date{2014}}

\section{Scholarships}
\entry{EMBL PhD fellowship}

\end{document}
