\documentclass{klmr-cv}

\setotherlanguage{french}

\author{Konrad Rudolph}
\email{konrad.rudolph@gmail.com}
\website{klmr.me}

\newcommand*\csharp{C\#}
\newcommand*\cpp{C++}

\begin{document}

\maketitle

\sidebar

\section{Degrees}

\subsection{University of Cambridge}

\date{2016}
\item{\type{PhD}}

\subsection{Freie Universit√§t Berlin}

\date{2011}
\item{\type{Master of Science}}
\item{Bioinformatics}

\subsection{Freie Universit√§t Berlin}

\date{2008}
\item{\type{Bachelor of Science}}
\item{Major: bioinformatics}
\item{Minor: physics}

\subsection{\textfrench{Cit√© Scolaire Internationale de Lyon}}

\date{2003}
\item{\type{Abitur \& baccalaur√©at}}

\section{Profiles}

\item{GitHub: \href{http://github.com/klmr}{klmr}}
\item{ORCID: \href{http://orcid.org/0000-0002-9866-7051}{0000-0002-9866-7051}}
\item{\href{http://stackoverflow.com/users/1968/konrad-rudolph}{Stack Overflow}}

\section{Skills}

\subsection{Languages}

\item{Fluency in written and spoken English, German \& French}

\subsection{Analysis}

\item{RNA-seq, ChIP-seq, motif analysis, data science: data exploration,
    visualisation, modelling}

\subsection{Programming}

\item{\cpp, C, R, Python, Perl, Bash, Make, \csharp, Java, PHP,~‚Ä¶}

\subsection{Tools}

\item{version control, unit testing, reproducible research, literate
    programming, lexical/semantic analysis, Unix, Windows, Mac}

\subsection{Management experience}

\item{Software development team lead}
\item{EMBL PhD Symposium organising committee}
\item{EMBL Bioinformatics Workshop coordination}

\section{Presentations}

\subsection{Invited talks}

\date{2014}
\item{Cambridge Epigenetics Symposium}

\body

\section{Research}

\subsection{University of Cambridge ‚Äî¬†Gurdon Institute}

\date{2016--2017}
\item{\type{Stress-induced, cross-generational deregulation of transposable
    elements in mouse spermatozoa}}
\date{2016--2017}
\item{\type{Transposable element silencing via the nuclear RNAi pathway}}
\date{2016--2017}
\item{\type{Translational regulation via codon frequency changes induced by tRNA
    base modifications in starved animals}}
\date{2016--2017}
\item{\type{Cross-individual (\textit{in hivo}) epigenetic communication in
    honey bees mediated by protein-bound RNA in bee jelly}}
\item{Group leader: Prof Dr Eric Miska}

\subsection{EMBL-EBI}

\date{2015--2016}
\item{\type{Quantifying genome-wide expression data of repeat elements}}
\date{2014--2015}
\item{\type{Does interaction between codon usage and tRNA abundance in mammals
    control translational efficiency?}}
\date{2011--2014}
\item{\type{tRNA \& mRNA gene expression during mammalian development}}
\item{Group leader: Dr John Marioni}

\subsection{Freie Universit√§t Berlin}

\date{2010--2011}
\item{\type{Generic parallelisation of a sequence analysis library}}
\item{Supervisor: Prof Dr Knut Reinert}

\date{2009}
\item{\type{Implementing a toolbox for T-invariant analysis in Petri nets of
    biochemical pathways}}
\item{Supervisor: Prof Dr Ina Koch}

\date{2008}
\item{\type{Implementation of a read mapping tool based on the pigeon-hole
    principle}}
\item{Supervisor: Prof Dr Knut Reinert}

\section{Work}

\workitem{May 2016}{}{University of Cambridge}{Postdoctoral research
    associate}{}
\workitem{Oct 2015}{Mar 2016}{EMBL-EBI}{Postdoctoral fellow}{}
\workitem{Oct 2011}{Oct 2015}{EMBL-EBI}{Predoctoral fellow}{}
\workitem{Jun 2011}{Aug 2011}{Independent consultancy}{Developer}{Integration of
    FPGA kernel with \cpp{} library}
\workitem{Oct 2008}{Feb 2009}{Illumina Inc.}{Research associate
    (intern)}{Implementation of short-read mapping on GPGPUs with Nvidia CUDA}
\workitem{2008}{2011}{Freie Universit√§t Berlin}{Tutor}{}
\workitem{Jan 2007}{Dec 2007}{ITosa}{Full-stack developer}{\csharp{} Windows
    application; PHP/JavaScript/HTML web application \& database frontend}

\section{Teaching}

\workitem{2017}{2017}{University of Cambridge}{smallRNA-seq}{}
\workitem{2017}{2017}{University of Cambridge}{NST Part II BBS Bioinformatics
    minor}{}
\workitem{2011}{2015}{EMBL-EBI}{Bioinformatics workshop}{Bash scripting, Unix,
    Git, R, \LaTeX{}}
\workitem{2013}{2015}{University of Cambridge}{Next generation sequencing
    course}{High-throughput sequencing techniques, with focus on ChIP-seq \&
    RNA-seq}
\workitem{2017}{2017}{University of Cambridge}{smallRNA sequencing}{}
\workitem{2008}{2011}{Freie Universit√§t Berlin}{Algorithms 101}{}
\workitem{2008}{2011}{Freie Universit√§t Berlin}{Algorithms 102}{}
\workitem{2008}{2011}{Freie Universit√§t Berlin}{Algorithms in bioinformatics}{}
\workitem{2008}{2011}{Freie Universit√§t Berlin}{Database systems}{}
\workitem{2008}{2011}{Freie Universit√§t Berlin}{\cpp{}}{}

\section{Selected publications}

% Bib(La)TeX does not cope at all with shared first authors (what the heck!).
% Hence we simply write the references here directly. ü§∑  ¬Ø\_(„ÉÑ)_/¬Ø

\begin{enumerate}
    \listitem Bianca M Schmitt*, Konrad L M Rudolph*, Panagiota Karagianni,
        Nuno A Fonseca, Robert J White, Iannis Talianidis, Duncan T Odom,
        John C Marioni, Claudia Kutter. ‚ÄúHigh-resolution mapping of
        transcriptional dynamics across tissue development reveals a stable
        mRNA--tRNA interface.‚Äù \textit{Genome Research}, August 2014,
        \href{http://dx.doi.org/10.1101/gr.176784.114}{DOI:
        10.1101/gr.176784.114}.

    \listitem Konrad L M Rudolph*, Bianca M Schmitt*, Diego Villar, Robert J
        White, John C Marioni, Claudia Kutter, Duncan T Odom. ‚ÄúCodon-driven
        translational efficiency is stable across diverse mammalian cell
        states.‚Äù \textit{PLOS Genetics}, May 2016,
        \href{http://dx.doi.org/10.1371/journal.pgen.1006024}{DOI:
        10.1371/journal.pgen.1006024}.
\end{enumerate}

\section{Scholarships}

\item{EMBL PhD fellowship}

\end{document}
