\documentclass{klmr-cv}

\setotherlanguage{french}

\author{Konrad Rudolph}
\email{konrad.rudolph@gmail.com}
\website{klmr.me}

\newcommand*\csharp{C\#}
\newcommand*\cpp{C++}
\newcommand*\macOS{\mbox{macOS}}

\begin{document}

\maketitle

\begin{sidebar}
    \section{Degrees}

    \subsection{University of Cambridge\date{2016}}
    \entry{PhD}

    \subsection{Freie Universit√§t Berlin\date{2011}}
    \entry{Master of Science}
    \details{Bioinformatics}

    \subsection{Freie Universit√§t Berlin\date{2008}}
    \entry{Bachelor of Science}
    \details{Major: bioinformatics\\Minor: physics}

    \section{Profiles}

    \subsection{GitHub}
    \details{\href{http://github.com/klmr}{klmr}}

    \subsection{ORCID}
    \details{\href{http://orcid.org/0000-0002-9866-7051}{0000-0002-9866-7051}}

    \subsection{Stack Overflow}
    \details{\href{http://stackoverflow.com/users/1968/konrad-rudolph}{Konrad Rudolph}}

    \section{Skills}

    \subsection{Languages}
    \details{Fluency in written \& spoken English, German, and French}

    \subsection{Analysis}
    \details{%
        \begin{minilist}
            \item Data exploration
            \item data visualisation
            \item statistical modelling
            \item RNA-seq
            \item ChIP-seq
            \item motif analysis
        \end{minilist}}

    \subsection{Programming}
    \details{%
        \begin{minilist}
            \item \cpp{}
            \item C
            \item R
            \item Python
            \item Perl
            \item Bash
            \item Make
            \item \csharp{}
            \item Java
            \item PHP
            \item ‚Ä¶
        \end{minilist}}

    \subsection{Tools}
    \details{%
        \begin{minilist}
            \item Version control
            \item unit testing
            \item reproducible research
            \item literate programming
            \item lexical/semantic analysis
            \item Unix
            \item Windows
            \item \macOS{}
        \end{minilist}}

    \subsection{Management}
    \details{%
        \begin{minilist}
            \item Software development team lead
            \item EMBL PhD Symposium organising committee
            \item EMBL Bioinformatics Workshop coordination
        \end{minilist}}
\end{sidebar}

\section{Research}

\subsection{Wellcome Trust CRUK Gurdon Institute\date{2016--2017}}

\entry{Stress-induced, cross-generational deregulation of transposable elements
    in mouse spermatozoa}
\entry{Transposable element silencing via the nuclear RNAi pathway}
\entry{Translational regulation via codon frequency changes induced by tRNA
    wobble base modification in starved animals}
\entry{Cross-individual (\textit{in hivo}) epigenetic communication in honey
bees mediated by protein-bound RNA in bee jelly}
\details{Group leader: Prof Dr Eric Miska}

\subsection{EMBL-EBI\date{2011--2016}}
\entry{Quantifying genome-wide expression data of repeat elements}

\entry{Interaction between codon usage and tRNA abundance in mammals to control
    translational efficiency}

\entry{tRNA \& mRNA gene expression during mammalian development}
\details{Group leader: Dr John Marioni}

\subsection{Freie Universit√§t Berlin\date{2010--2011}}
\entry{Generic parallelisation of a sequence analysis library}
\details{Supervisor: Prof Dr Knut Reinert}

\subsection{Freie Universit√§t Berlin\date{2009}}
\entry{Implementation of a tool for T-invariant analysis in Petri nets of
    biochemical pathways}
\details{Group leader: Prof Dr Ina Koch}

\subsection{Freie Universit√§t Berlin\date{2008}}
\entry{Implementation of a read mapping tool based on the pigeon-hole principle}
\details{Supervisor: Prof Dr Knut Reinert}

\section{Work}

\subsection{University of Cambridge\date{May 2016--}}
\entry{Postdoctoral research associate}

\subsection{EMBL-EBI\date{Oct 2015--Mar 2016}}
\entry{Postdoctoral fellow}

\subsection{EMBL-EBI\date{Oct 2011--Sep 2015}}
\entry{Predoctoral fellow}

\subsection{Independent consultancy\date{Jun 2011--Aug 2011}}
\entry{Developer}
\details{Integration of FPGA kernel with \cpp{} library}

\subsection{Illumina Inc.\date{Oct 2008--Jan 2009}}
\entry{Research associate (intern)}
\details{Implementation of short-read mapping on GPGPUs with Nvidia CUDA}

\subsection{Freie Universit√§t Berlin\date{2008--2011}}
\entry{Tutor}

\subsection{ITosa\date{Jan 2007--Dec 2007}}
\entry{Full stack developer}
\details{\csharp{} Windows application; PHP/JavaScript/HTML web application \&
    database frontend}

\section{Teaching}

\subsection{University of Cambridge}
\entry{smallRNA sequencing\date{2017}}
\entry{NST Part II BBS Bioinformatics minor\date{2017}}
\entry{Next generation sequencing course\date{2013--2015}}
\details{High-throughput sequencing techniques, with focus on ChIP-seq \&
    RNA-seq}

\subsection{EMBL-EBI\date{2011--2015}}
\entry{Bioinformatics workshop}
\details{%
    \begin{minilist}
        \item Bash scripting
        \item Unix
        \item Git
        \item R
        \item \LaTeX{}
    \end{minilist}}

\subsection{Freie Universit√§t Berlin\date{2008--2011}}
\details{%
    \begin{minilist}
        \item Algorithms 101
        \item Algorithms 102
        \item Algorithms in bioinformatics
        \item Database systems
        \item \cpp{}
    \end{minilist}}

\section{Selected publications}

% Bib(La)TeX does not cope at all with shared first authors (what the heck!).
% Hence we simply write the references here directly. ü§∑  ¬Ø\_(„ÉÑ)_/¬Ø

\details{%
\begin{enumerate}
    \item Bianca M Schmitt*, Konrad L M Rudolph*, Panagiota Karagianni,
        Nuno A Fonseca, Robert J White, Iannis Talianidis, Duncan T Odom,
        John C Marioni, Claudia Kutter. ‚ÄúHigh-resolution mapping of
        transcriptional dynamics across tissue development reveals a stable
        mRNA--tRNA interface.‚Äù \textit{Genome Research}, August 2014,
        \href{http://dx.doi.org/10.1101/gr.176784.114}{DOI:
        10.1101/gr.176784.114}.

    \item Konrad L M Rudolph*, Bianca M Schmitt*, Diego Villar, Robert J
        White, John C Marioni, Claudia Kutter, Duncan T Odom. ‚ÄúCodon-driven
        translational efficiency is stable across diverse mammalian cell
        states.‚Äù \textit{PLOS Genetics}, May 2016,
        \href{http://dx.doi.org/10.1371/journal.pgen.1006024}{DOI:
        10.1371/journal.pgen.1006024}.
\end{enumerate}
}

\section{Presentations}

\subsection{Invited talks}
\details{Cambridge Epigenetics Symposium\date{2014}}

\section{Scholarships}
\details{EMBL PhD fellowship}

\end{document}
