\documentclass{klmr-cv}

\setotherlanguage{french}

\author{Konrad Rudolph}
\email{konrad.rudolph@gmail.com}
\website{klmr.me}

\newcommand*\csharp{C\#}
\newcommand*\cpp{C++}

\begin{document}

\maketitle

\sidebar

\section{Current position}

\subsection{EMBL-EBI}

\date{2015--2016}
\item{\type{Postdoctoral fellow}}

\section{Degrees}

\subsection{University of Cambridge}

\date{2015}
\item{\type{PhD}}

\subsection{Freie Universität Berlin}

\date{2011}
\item{\type{Master of Science}}
\item{Bioinformatics}

\subsection{Freie Universität Berlin}

\date{2008}
\item{\type{Bachelor of Science}}
\item{Major: bioinformatics}
\item{Minor: physics}

\subsection{\textfrench{Cité Scolaire Internationale de Lyon}}

\date{2003}
\item{\type{Abitur \& baccalauréat}}

\section{Profiles}

\item{GitHub: \href{http://github.com/klmr}{klmr}}
\item{ORCID: \href{http://orcid.org/0000-0002-9866-7051}{0000-0002-9866-7051}}
\item{\href{http://stackoverflow.com/users/1968/konrad-rudolph}{Stack Overflow}}

\section{Skills}

\subsection{Languages}

\item{Fluency in written and spoken English, German \& French}

\subsection{Analysis}

\item{RNA-seq, ChIP-seq, motif analysis, data exploration}

\subsection{Programming}

\item{\cpp, C, R, Python, Perl, Bash, Make, \csharp, Java, PHP,~…}

\subsection{Tools}

\item{version control, unit testing, reproducible research, literate
    programming, lexical/semantic analysis, Unix, Windows, Mac}

\subsection{Administrative experience}

\item{EMBL PhD Symposium organising committee}
\item{EMBL Bioinformatics Workshop coordination}

\body

\section{Research}

\subsection{EMBL-EBI}

\date{2015--2016}
\item{\type{Quantifying genome-wide expression data of repeat elements}}
\date{2014--2015}
\item{\type{Does interaction between codon usage and tRNA abundance in mammals
    control translational efficiency?}}
\date{2011--2014}
\item{\type{tRNA \& mRNA gene expression during mammalian development}}
\item{Group leader: Dr John Marioni}

\subsection{Freie Universität Berlin}

\date{2010--2011}
\item{\type{Generic parallelisation of a sequence analysis library}}
\item{Supervisor: Prof Dr Knut Reinert}

\date{2009}
\item{\type{Implementing a toolbox for T-invariant analysis in Petri nets of
    biochemical pathways}}
\item{Supervisor: Prof Dr Ina Koch}

\date{2008}
\item{\type{Implementation of a read mapping tool based on the pigeon-hole
    principle}}
\item{Supervisor: Prof Dr Knut Reinert}

\section{Teaching}

\subsection{EMBL-EBI}

\date{2011--2015}
\item{\type{Bioinformatics workshop}}
\item{Bash scripting, Unix, Git, R, \LaTeX}

\subsection{University of Cambridge}

\date{2013--2015}
\item{\type{Next generation sequencing course}}
\item{High-throughput sequencing techniques, with focus on ChIP-seq \& RNA-seq}

\subsection{Freie Universität Berlin}

\date{2008--2011}
\item{\type{Tutor}}
\item{Algorithms 101/102, Algorithms in bioinformatics, Database systems, \cpp}

\section{Industry}

\subsection{Illumina, Inc.}

\date{Oct 2008--Feb 2009}
\item{\type{Research associate (intern)}}
\item{Implementation of short-read mapping on GPGPUs with Nvidia CUDA}

\subsection{ITosa}

\date{Jan 2007--Jan 2008}
\item{\type{Full-stack developer}}
\item{\csharp{} smart client; PHP/JavaScript/HTML web application \& database
    frontend}

\section{Selected publications}

% Bib(La)TeX does not cope at all with shared first authors (what the heck!).
% Hence we simply write the references here directly.

\begin{enumerate}
    \listitem Bianca M Schmitt*, Konrad L M Rudolph*, Panagiota Karagianni,
        Nuno A Fonseca, Robert J White, Iannis Talianidis, Duncan T Odom,
        John C Marioni, Claudia Kutter. “High-resolution mapping of
        transcriptional dynamics across tissue development reveals a stable
        mRNA--tRNA interface.” \textit{Genome Research}, August 2014.
        \href{http://dx.doi.org/10.1101/gr.176784.114}{doi:10.1101/gr.176784.114}.
\end{enumerate}

\section{Presentations}

\subsection{Invited talks}

\date{2014}
\item{Cambridge Epigenetics Symposium}

\end{document}
